### DISCUSSION TITLE
Stable API usage?

### DISCUSSION THREAD

**Discussion by yajo** [thombashi/sqlitebiter#94] [Q&A] [answered]
Hello!

I'm trying to use this project as a library inside another.

I noticed that in the docs, there's only mention of CLI usage. I also noticed that most code is inside \_underscored modules.

This gives me the feel that the only stable interface with sqlitebiter is CLI. If that's the case, it's perfectly OK. But I'd love to know if there's a way to use it as a library. At least in a way that is relatively future-proof.

Thanks!

**Comment by thombashi** [comment, marked as answer]
@yajo 
> This gives me the feel that the only stable interface with sqlitebiter is CLI.

That is correct.
Unfortunately, there are no public interfaces to use `sqlitebiter` as a library.
